front-end/src/app/shared/layout/header/header.component.css

```diff
@@ -54,31 +50,66 @@ body {
   background: #fff;
 }
 
-#items-van a,
-a:hover,
-a:focus {
-  transition: all 0.3;
+#sidebar-wrapper {
+  position: fixed;
+  z-index: 1;
+  width: 15rem;
+  min-height: 100vh;
+  background: #00417c;
 }
 
 #page-content-wrapper {
   min-width: 100vw;
+  margin-left: 15rem;
+}
+
+#content {
+  padding-top: 50px;
+  padding-right: 15rem;
 }
 
-#wrapper.toggled #sidebar-wrapper {
+.toggled-side {
+  margin-left: -15rem !important;
+}
+
+.toogled-page {
   margin-left: 0;
 }
 
-@media (min-width: 768px) {
+.toggled-nav {
+  left: 0 !important;
+}
+
+.toggled-content{
+  padding-right: 0 !important;
+}
+
+@media (max-width: 768px) {
   #sidebar-wrapper {
-    margin-left: 0;
+    margin-left: -15rem;
+  }
+  #id-navbar {
+    left: 0;
   }
 
   #page-content-wrapper {
-    min-width: 0;
-    width: 100%;
+    margin-left: 0;
   }
 
-  #wrapper.toggled #sidebar-wrapper {
-    margin-left: -15rem;
+  .toggled-side {
+    margin-left: 15rem !important;
+  }
+
+  .toogled-page {
+    margin-left: 15rem;
+  }
+
+  .toggled-nav {
+    left: 15rem !important;
+  }
+
+  /*Contenido movil*/
```

Comment:
Please write comments in English so we can maintain consistency throughout the project